src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/health/health.component.ts

```diff
@@ -148,6 +148,7 @@ export class HealthComponent implements OnInit, OnDestroy {
 
   preparePgStatus(chart, data) {
     const categoryPgAmount = {};
+    let totalPgs = 0;
```

Comment:
I've done exactly [the same for the downstream stuff](https://github.com/rhcs-dashboard/ceph/pull/83/files#diff-5ce71566f5c9c834be00267dd292e1d4R154) (but my goal was not exactly to fix this, but have only percentages shown in labels).